action.yaml

```diff
@@ -68,7 +67,6 @@ inputs:
   hide-progress:
     description: 'hide progress output'
     required: false
-    default: 'true'
```

Comment:
I'm not sure I understand the upstream default here:

```sh
$ trivy i --help | grep -i progress
   --no-progress                    suppress progress bar (default: false) [$TRIVY_NO_PROGRESS]
```

Is the default to display the progress bar if `--no-progress` is omitted? In that case, we would be breaking BC in the action, right?

If not, then fine ;) Just remember to update README as well.

Reply:
Good catch - I had to make this change because the default behaviour of the action is to add `--no-progress` like you mentioned but the `sbom` mode in Trivy doesn't have this flag just yet. 

```
trivy sbom --help
NAME:
   trivy sbom - generate SBOM for an artifact

USAGE:
   trivy sbom [command options] ARTIFACT

DESCRIPTION:
   ARTIFACT can be a container image, file path/directory, git repository or container image archive. See examples.

OPTIONS:
   --artifact-type value, --type value  input artifact type (image, fs, repo, archive) (default: "image") [$TRIVY_ARTIFACT_TYPE]
   --clear-cache, -c                    clear image caches without scanning (default: false) [$TRIVY_CLEAR_CACHE]
   --db-repository value                OCI repository to retrieve trivy-db from (default: "ghcr.io/aquasecurity/trivy-db") [$TRIVY_DB_REPOSITORY]
   --ignorefile value                   specify .trivyignore file (default: ".trivyignore") [$TRIVY_IGNOREFILE]
   --insecure                           allow insecure server connections when using SSL (default: false) [$TRIVY_INSECURE]
   --offline-scan                       do not issue API requests to identify dependencies (default: false) [$TRIVY_OFFLINE_SCAN]
   --output value, -o value             output file name [$TRIVY_OUTPUT]
   --sbom-format value, --format value  SBOM format (cyclonedx, spdx, spdx-json, github) (default: "cyclonedx") [$TRIVY_SBOM_FORMAT]
   --severity value, -s value           severities of vulnerabilities to be displayed (comma separated) (default: "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL") [$TRIVY_SEVERITY]
   --skip-dirs value                    specify the directories where the traversal is skipped  (accepts multiple inputs) [$TRIVY_SKIP_DIRS]
   --skip-files value                   specify the file paths to skip traversal                (accepts multiple inputs) [$TRIVY_SKIP_FILES]
   --timeout value                      timeout (default: 5m0s) [$TRIVY_TIMEOUT]
   
EXAMPLES:
  - image scanning:
      $ trivy sbom alpine:3.15

  - filesystem scanning:
      $ trivy sbom --artifact-type fs /path/to/myapp

  - git repository scanning:
      $ trivy sbom --artifact-type repo github.com/aquasecurity/trivy-ci-test

  - image archive scanning:
      $ trivy sbom --artifact-type archive ./alpine.tar
```

It won't break backwards compatibility in the sense that the only behavioural change is extra output in stdout during CI scans. I think it is acceptable as functionally theres no change. 

cc @knqyf263 - can we add a `no-progress` at a global level for Trivy so that all scan options can have it? Today it is missing in `sbom` as mentioned above.